sql-plugin/src/main/scala/com/nvidia/spark/rapids/literals.scala

```diff
@@ -329,6 +490,7 @@ object GpuLiteral {
     val cpuLiteral = Literal.default(dataType)
     GpuLiteral(cpuLiteral.value, cpuLiteral.dataType)
   }
+
```

Comment:
empty line?

Reply:
yes, easy to distinguish the ends of the function and the `GpuLiteral ` object.